ouroboros-consensus-test/src/Test/Util/FS/Sim/MockFS.hs

```diff
@@ -48,16 +44,15 @@ module Test.Util.FS.Sim.MockFS (
   , removeFile
   , renameFile
     -- * Exported for the benefit of tests only
-  , HandleState
-    -- opaque
-  , OpenHandleState
-    -- opaque
-  , ClosedHandleState
-    -- opaque
-  , FilePtr
-    -- opaque
   , Files
   , mockFiles
+    -- * opaque
```

Comment:
Let's use `**` since these were all in the "Exported for the benefit of tests only" section

Reply:
Actually, let's just remove `-- opaque` from the ones that are already in the testing section. We don't need the extra info about those, I don't think.

So just `MockFS` and `HandleMock` in the next section, I think.